tests/test_params.py

```diff
@@ -0,0 +1,63 @@
+from nose.tools import (
```

Comment:
We are getting away from `nose`. All new tests should not be implemented using `pytest`